entries/s/stripe.com.json

```diff
@@ -5,7 +5,7 @@
     "tfa": [
       "sms",
       "totp",
-      "custom-hardware"
```

Comment:
They support _Windows Hello_ and _Touch ID_ as well - maybe we need both `custom-hardware` and ` u2f` here?